src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -230,9 +230,8 @@ if ($components) {
     if (!class_exists('SymfonyBlacklistSimplePhpunit', false)) {
         class SymfonyBlacklistSimplePhpunit {}
     }
-    array_splice($argv, 1, 0, array('--colors=always'));
-    $_SERVER['argv'] = $argv;
-    $_SERVER['argc'] = ++$argc;
+    array_splice($_SERVER['argv'], 1, 0, array('--colors=always'));
```

Comment:
ok, in fact, this change introduces a bug
both `$_SERVER['argv']` and `$argv` must be updated
this breaks the second